model/GameModel.ts

```diff
@@ -22,18 +103,91 @@ export class GameModel {
         return this._imageUrl;
     }
 
-    get birthYear(): number {
-        return this._birthYear;
+    get birth(): Hint<Date> {
+        return this._birth;
+    }
+
+    get death(): Hint<Date> {
+        return this._death;
+    }
+
+    get occupation(): Hint<string> {
+        return this._occupation;
+    }
+
+    get citizenship(): Hint<string> {
+        return this._citizenship;
+    }
+
+    get initials(): Hint<string> {
+        return this._initials;
+    }
+
+    get paragraph1(): Hint<string> {
+        return this._paragraph1;
     }
 
     get blur(): number {
         return this._blur;
     }
 
+    get curGuess() : string {
+        return this._curGuess;
+    }
+
+    get nbGuesses() : number {
+        return this._nbGuesses;
+    }
+
+    get end() : boolean {
+        return this._end;
+    }
+
+    get win() : boolean {
+        return this._win;
+    }
+
     set blur(value: number) {
         if (value < 0 || value > 7) {
             throw new Error("Blur must be between 0 and 7");
         }
         this._blur = value;
     }
-}
\ No newline at end of file
+}
+
+
+/**
+ * this class represents a hint, the value being the core of the hint
+ * revealed tells us if the hint has been revealed yet while level gives us
+ * the level corresponding to the hint (1 being a small hint and 3 a huge hint)
+ */
+export class Hint <T> {
+    private _level : number;
```

Comment:
Wy do you use private values here ?

Reply:
because these values shouldn't be changed, and should only be accessed through the getters, basically it is protection